src/common/utils/recommend.ts

```diff
@@ -92,9 +92,13 @@ export function getFedLevel(projection: Projection): FedLevel | null {
 
   if (weeklyCasesPer100k > 100 && positiveTestRate > 0.1) {
     return FedLevel.RED;
+  } else if (weeklyCasesPer100k > 100 || positiveTestRate > 0.1) {
+    return FedLevel.YELLOW;
```

Comment:
Hrm, isn't supposed to be one "red condition" (covered by your `if`) but also one "yellow condition" (not covered by your `if`)?

So by the fed rules, I think maybe positiveTestRate > 0.1 but weeklyCasesPer100k < 10 would be green, since it's only meeting 1 red condition and 0 yellow conditions.

But I may be confused! I haven't looked at the full fed guidelines.

Reply:
ah no i just re-visited the fed rules and i think you're right. i wasn't factoring in that the non-red condition still needs to be yellow (and not green).